generatedocs.py

```diff
@@ -3,8 +3,25 @@
 import json # reading the input file
 import os # file paths
 import re # regex for searching for assets files
+import shutil # copytree, rmtree
 from tqdm import tqdm # progress bar
 
+##########################################################################################
+#                                     CUSTOMIZATION                                      #
+##########################################################################################
+# This is the branch on the OpenSpace repository from which the documentation will be
+# built. Change this to a different branch to try a local branch before committing.
+# OBS: No other value than `master` should ever be committed to the master branch of the
+#      docs repository
+OPENSPACE_BRANCH = "master"
+
+# If this value is specified, instead of cloning OpenSpace from the main repository using
+# the branch provided above, instead use a local copy of the repository.
+# OBS: No other value than the empty string should ever be committed to the master branch
+#      of the docs repository
+LOCAL_OPENSPACE_FOLDER = ""
+
+
```

Comment:
We could place these as arguments to the `generate_docs` function, which would mean that these variables could be stored in the `conf.py`. I would think our goal is that the `conf.py` is the only file in this repo except for `.md` files that should be edited on a regular basis so could be a way to achieve that? If a new user wants to build this repo I would think that the `conf.py` is the first file they would open